src/renderer/components/TrackList/TrackList.vue

```diff
@@ -125,7 +129,12 @@ export default {
                 this.$toast.message('已退出私人 FM');
                 this.activateRadio(false);
             }
-            this.playTrack(index);
+            if (this.autoAddToPlaylist) {
```

Comment:
May add `'settings'` to `mapState above`

```js
...mapState(['ui', 'user', 'playlist', 'settings']),
```

and use

```js
this.settings.autoAddToPlaylist
```

here, to match the style of existing codebase ...